### ISSUE TITLE
Sear knockback

### ISSUE THREAD

**Issue by ContraMods** [GnomeModder/ChefMod#5]
Sear's knockback is EXTREMELY high to the point ANY living creature gets tossed like in Super Smash Bros.

**Comment by Moffein** [comment]
This is an intended feature. Its knockback scales based on mass, so that it knocks everything back in a consistent manner for the most part. This is to give CHEF the option to knock things back into his oil pools or off of cliffs.

**Comment by ContraMods** [comment]
Yes but when you have mods like SizeManagerPlus, the mass increases proportional to the monster's size, hence increasing knockback greatly.
